gem/context_processors.py

```diff
@@ -40,6 +40,7 @@ def compress_settings(request):
 
     if settings.USE_OIDC_AUTHENTICATION:
         site = request.site
+        language = getattr(request, 'LANGUAGE_CODE', 'en')
```

Comment:
Do we want it to always be english or should we not set a value in settings, like the theme

Reply:
Good point